docs/collaboration.md

```diff
@@ -0,0 +1,105 @@
+# Collaborative Publishing with Posit Publisher
+
+[Posit Publisher](https://github.com/posit-dev/publisher)—the extension to
+publish content to Posit Connect using
+[Visual Studio Code](https://code.visualstudio.com/) or
+[Positron](https://positron.posit.co/)—enables seemless collaboration. This
+workflow guide explains how to collaborate with others on a deployed piece of
+content.
+
+## Deployment and Configuration files
+
+Posit Publisher creates and utilizes two files for deploying content. Both are
+inside of the `.posit/publish` directory in the same directory as
+your entrypoint file.
+
+#### The Configuration file
+
+The Configuration file describes how the content will be deployed to Posit
+Connect.
+
+An example of a Configuration file is `/.posit/publish/content-title-061E.toml`.
+
+For more details on the Configuration file see the [Configuration file reference](./configuration.md).
+
+#### The Deployment file
+
+The Deployment file contains details on where the content was deployed and its
+state. Deployment files are automatically generated by Posit Publisher,
+updated during deploys, and not meant to be edited.
+
+An example of a Deployment file is
+`/.posit/publish/deployments/deployment-58A3.toml`.
+
+Utilizing both of these files, alongside a credential and collaborator access
+on Posit Connect, you and your collaborators can update and configure a single
+piece of content.
+
+## Collaborator access
+
+The original creator of a piece of content on Posit Connect needs to give others
+collaborator access to update the content.
+
+See the [Posit Connect User Guide - Access list settings](https://docs.posit.co/connect/user/content-settings/#set-collaborators)
+documentation for more information on giving users access to update a piece of
+content.
+
+## Source control setup
+
+Deployment and Configuration files found in `./posit/publish` are intended
+to be committed into source repositories.
+
+Once tracked in source control, anyone with access can easily collaborate on the
+same piece of content.
+
+After a deploy be sure to commit any changes to the Deployment and Configuration
+files. Keeping them in sync with the content on Posit Connect will ensure that
+collaborators have the most up-to-date details to continue where others left
+off.
+
+## Content bundle setup
+
+If you and your collaborators aren't using source control, a content bundle
+containing the source code and data for a piece of content can be downloaded
+from Posit Connect. Content bundles are created when content is published and
+they work great with Posit Publisher.
+
+When deploying, include the Deployment and Configuration files so others can
+use them to collaborate after they download the content bundle.
```

Comment:
Maybe this should be higher up? This is the big "if you do this things will generally just work" bit. Like if someone reading this section could read and remember a single sentence it's this one yeah?

Reply:
Great point. I put the tip to commit the files right at the start of the source control section - I should do the same here.